libs/features/spectrum-web-components/dist/icons-ui.js

```diff
@@ -2,6 +2,7 @@
 /* Generated by Milo */
 
 import{html as K}from"./base.js";import{IconBase as N}from"./icon.js";var T,o=function(t,...r){return T?T(t,...r):r.reduce((e,G,J)=>e+G+t[J+1],t[0])},s=t=>{T=t};var L=({width:t=24,height:r=24,title:e="Arrow75"}={})=>o`<svg
+    aria-hidden="true"
```

Comment:
Attributes already exist, no changes needed.

Reply:
Removed it 👍